sigar_linux_common_test.go

```diff
@@ -0,0 +1,89 @@
+		"portal /run/user/1000/doc fuse.portal rw,nosuid,nodev,relatime,user_id=1000,group_id=1000 0 0",
+		"gvfsd-fuse /run/user/1000/gvfs fuse.gvfsd-fuse rw,nosuid,nodev,relatime,user_id=1000,group_id=1000 0 0",
+		"/dev/loop75 /snap/discord/225 squashfs ro,nodev,relatime,errors=continue,threads=single 0 0",
+	}
+
+	expected_list := []sigar.FileSystem{}
```

Comment:
Nit. Code conventions.
```suggestion
	expectedList := []sigar.FileSystem{}
```